mod-consortia/src/main/resources/thunderjet/mod-consortia/consortia-junit.feature

```diff
@@ -75,61 +75,89 @@ Feature: mod-consortia integration tests
+#    * call read('classpath:common-consortia/tenant-and-local-admin-setup.feature@SetupTenant') { tenant: '#(universityTenant)', admin: '#(universityUser1)'}
+#    * call read('classpath:common-consortia/tenant-and-local-admin-setup.feature@SetupTenant') { tenant: '#(collegeTenant)', admin: '#(collegeUser1)'}
+#
+#    # create users in all tenants
+#    * call read('classpath:common-consortia/create-users.feature@CreateUsers')
+#
+#    # add 'consortia.all' (for consortia management) and 'tags.all' (for publish coordinator tests) permissions to main users
+#    * call login consortiaAdmin
+#    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+#
+#    * call login universityUser1
+#    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+#
+#    * call login collegeUser1
+#    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+#
+#  Scenario: Consortium api tests
+#    * call read('features/consortium.feature')
+#
+#  Scenario: Tenant api tests
+#    * call read('features/tenant.feature')
+#
+#  Scenario: verify and setup 'consortiaAdmin' for all tenants
+#    * call read('features/consortia-admin-verification-and-setup.feature')
+#
+#  Scenario: verify 'consortia-system-user' in all tenants
+#    * call read('features/consortia-system-users-verification.feature')
+#
+#  Scenario: User-Tenant associations api tests
+#    * call read('features/user-tenant-associations.feature')
+#
+#  Scenario: verify users with shadow or patron types not processed by consortia pipeline
+#    * call read('features/consortia-skip-not-required-user-types.feature')
+#
+#  Scenario: verify user update scenarios
+#    * call read('features/consortia-user-update.feature')
+#
+#  Scenario: verify user type update scenarios
+#    * call read('features/consortia-user-type-update.feature')
+#
+#  Scenario: Publish coordinator tests
+#    * call read('features/publish-coordinator.feature')
+#
+#  Scenario: Sharing Instances api tests
+#    * call read('features/sharing-instance.feature')
+#
+#  Scenario: Sharing Settings api tests
+#    * call read('features/sharing-setting.feature')
+#
+#  Scenario: Sharing Patron Groups Settings api tests
+#    * call read('features/sharing-patron-groups-setting.feature')
 
   Scenario: Destroy created ['central', 'university', 'college'] tenants
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(universityTenant)'}
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(collegeTenant)'}
-    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: '#(centralTenant)'}
+    * call read('classpath:common-consortia/initData.feature@DeleteTenant') { tenant: 'central1719409122040'}
```

Comment:
Can this be stuffed into some array and for-looped?

Reply:
Or is this a temporary cleanup that got accidentally committed?

Reply:
Didn't expect this changes, I think it come from older comments, removed